Docs/source/building/cori.rst

```diff
@@ -51,6 +51,40 @@ In order to compile for the **Knight's Landing (KNL) architecture**:
         module swap PrgEnv-intel PrgEnv-gnu
         make -j 16 COMP=gnu
 
+GPU Build
+---------
+
+To compile on the experimental GPU nodes on Cori, you first need to purge
+your modules, most of which won't work on the GPU nodes.
+
+   ::
+
+	module purge
```

Comment:
Could you indent the above?

Reply:
Not sure what you mean - it looks the same as all the others to me.

Reply:
Here is how it looks in the "Files changed" panel:
<img width="541" alt="Screen Shot 2019-05-24 at 2 35 14 PM" src="https://user-images.githubusercontent.com/6685781/58357922-8fa58f00-7e31-11e9-9bd6-3ceaf90fc33d.png">
Is that the same for you?

Reply:
No, it looks like 

```
your modules, most of which won't work on the GPU nodes.

    ::

 	module purge
```

to me.